spectral_cube/spectral_cube.py

```diff
@@ -254,10 +254,15 @@ def _try_load_beam(self, header):
         try:
             self.beam = Beam.from_fits_header(header)
             self._meta['beam'] = self.beam
+            self.beam_area_pixels = self._beam_area_pixels
         except:
             warnings.warn("Could not parse beam information from header.")
 
     @property
+    def _beam_area_pixels(self):
+        return (self.beam.sr/(wcs.utils.proj_plane_pixel_area(self.wcs)*u.deg**2)).to(u.dimensionless_unscaled).value
```

Comment:
my intent was to make this an immutable value, but it probably still is not because it's going to grab the value from the property... so, no, the property is pointless